servlet/src/main/scala/org/http4s/servlet/BlockingHttp4sServlet.scala

```diff
@@ -49,7 +50,7 @@ class BlockingHttp4sServlet[F[_]] private (
   ): Unit = {
     val result = F
       .defer {
-        val bodyWriter = servletIo.bodyWriter(servletResponse, dispatcher) _
+        val bodyWriter = servletIo.initWriter(servletResponse)
```

Comment:
I think the old version is what we want in the long run... we just have to give it a dummy dispatcher for now.  It becomes important when we get back to the fs2.Queue implementation.

Reply:
Up in the next PR. Thank you